app/javascript/react/src/containers/BookShowContainer.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import BookShowTile from '../components/BookShowTile'
+import ReviewTile from '../components/ReviewTile'
+
+
+class BookShowContainer extends React.Component {
+  constructor(props) {
+    super(props);
+      this.state = {
+        book: {},
+        reviews: []
+      }
+  }
+
+  componentDidMount() {
+    // let bookId = this.props.params.id;
+    fetch(`/api/v1/books/6`)
```

Comment:
This needs to be changed to be dynamic (before I'll merge it)! The `showContainer` looks really good otherwise!